client/src/main/java/com/paypal/selion/internal/reports/runtimereport/MethodInfo.java

```diff
@@ -52,7 +52,7 @@
     private String exception;
     private String stacktrace;
     private List<LogInfo> logs;
-    private transient ITestResult result;
+    private final transient ITestResult result;
```

Comment:
I think the rule in codacy about making all fields "final" can be a bit of a blunt instrument.. As I look a this though it certainly isn't "wrong" it just looks really unusual to me.  The gson \@Expose format certainly helps to clarify what's happening for developers.